static/app/utils/featureObserver.ts

```diff
@@ -1,4 +1,4 @@
-import type {FeatureFlagContext} from '@sentry/types/build/types/context';
+import type {FeatureFlagContext} from '@sentry/core/build/types/types-hoist/context';
```

Comment:
Should it not be

```suggestion
import type {FeatureFlagContext} from '@sentry/core';
```

Reply:
I get an error importing directly. Looks like it's not exported in index

Reply:
Let's make sure this gets exported and updated.